types/Order/Order.d.ts

```diff
@@ -123,4 +124,12 @@ export declare class Order implements IOrder {
    * @returns {Promise<Irate[]>} The list of {@link IRate[]}.
    */
   static getRates(orderId: string): Promise<IRate[]>;
+
+  /**
+   *
+   * @param carriers a list of carriers to filter rates for.
+   * @param services a list of services to filter rates for.
+   * @returns {Promise<Rate>} The lowest {@link Rate}.
+   */
+  lowestRate(carriers?: string[], services?: string[]): IRate;
```

Comment:
Issue:
Hmmm... it says it returns a promise of `Rate`, but the type says it just returns `IRate`. Should this be `Promise<IRate>` instead?

Reply:
Ooo it's not a promise since we're just filtering a list. I'll fix the returns comment instead